packages/integrations/src/angular.ts

```diff
@@ -47,6 +47,8 @@ export class Angular implements Integration {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   public constructor(options: { angular?: any } = {}) {
+    logger.log('You are still using the Angular integration, consider moving to @sentry/angular');
```

Comment:
I didn't mean to just change the log type 😅 I was thinking about something like `if (!options.supressWarning)` in all deprecated integrations.

Reply:
Hmm, I am not sure if introducing an option is the right thing for this, I rather have a defensive logging for now and don't annoy people in prod. They should see it with `debug: true` but that's it then,

Reply:
Ok, let's add it once someone asks for it.